### PR TITLE
[opentelemetry] adding logs label, fixing ceph parsing

### PR DESCRIPTION
**Pull request by Kuckkuck**
- Fixed ceph log line parsing for resource.attributes["app.label.component"] == "cephobjectstores.ceph.rook.io"
- added label for log.type
- increase memory limit
- more Kubernetes labels from the Kubernetes metadata service

### CODE REVIEW

File: opentelemetry/chart/templates/logs-collector.yaml [hunk 1]
```diff
@@ -56,13 +56,19 @@ spec:
     receivers:
       journald:
         directory: /var/log/journal
+        operators:
+          - id: journal-label
+            type: add
+            field: attributes["log.type"]
+            value: "journald"
```
Comment (viennaa) [inline opentelemetry/chart/templates/logs-collector.yaml:63]: are the quotes necessary?

File: opentelemetry/chart/templates/logs-collector.yaml [hunk 2]
```diff
@@ -128,6 +134,9 @@ spec:
           - action: insert
             key: region
             value: "${region}"
+          - action: insert
+            key: log.type
+            value: "k8sevents"
```
Comment (viennaa) [inline opentelemetry/chart/templates/logs-collector.yaml:139]: same question :)

**Review by viennaa — COMMENTED** [review]
Small nit from my side.

File: opentelemetry/chart/templates/logs-collector.yaml (hunk 2 above)
Comment (Kuckkuck) [reply opentelemetry/chart/templates/logs-collector.yaml:139]: same answer ;-)

**Pull request closed without merging**